gtsam/hybrid/GaussianMixture.h

```diff
@@ -144,6 +144,26 @@ class GTSAM_EXPORT GaussianMixture
   /// Getter for the underlying Conditionals DecisionTree
   const Conditionals &conditionals();
 
+  /**
+   * @brief Compute error of the GaussianMixture as a tree.
+   *
+   * @param continuousVals The continuous VectorValues.
+   * @return AlgebraicDecisionTree<Key> A decision tree with corresponding keys
+   * as the factor but leaf values as the error.
```

Comment:
It should read "keys corresponding to the factors involved", or "__the same keys__ as the factor".

Since this is a GaussianMixture, I should be using "conditionals" and not "factors".